admin-dev/themes/new-theme/js/pages/product/services/combinations-service.ts

```diff
@@ -67,11 +77,11 @@ export default class CombinationsService {
   }
 
   /**
-   * @param {int} combinationId
+   * @param {number} combinationId
    *
    * @returns {Promise}
    */
-  deleteCombination(combinationId) {
+  deleteCombination(combinationId: number): JQuery.jqXHR<any> {
```

Comment:
```suggestion
  deleteCombination(combinationId: number): Promise<FetchResponse> {
```

This kind of typing seems simpler and more human-readable, maybe we could use it everywhere?

Reply:
We can't, as we use jQuery

Reply:
Oh right, and there is no equivalent with fetch using the `DELETE` method?